cloud/sts.go

```diff
@@ -50,6 +51,8 @@ type AssumeRoleOptions struct {
 	// DurationSeconds is an optional field of the duration of the role session.
 	// It defaults to 15 minutes.
 	DurationSeconds *int32
+	// Cache signals whether to cache the credentials.
+	Cache bool
```

Comment:
nit: slight preference for calling this `UseCache`/`CanCache`, since `opts.Cache` (using cache as a verb instead of noun) typically means to put something in a cache, whereas it's also being used to decide if we can retrieve from the cache.

Reply:
Switched to `CanCache`